static/js/containers/UserPage_test.js

```diff
@@ -233,6 +291,43 @@ describe("UserPage", function() {
         });
       });
 
+      it('should show the entries in resume order', () => {
```

Comment:
please change the references to 'resume order' in the `it()` statements here and above